lib/module.js

```diff
@@ -1,16 +1,23 @@
 'use strict';
 
 const NativeModule = require('native_module');
-const util = require('util');
+const assert = require('internal/assert');
+const debug = require('internal/debuglog')('module');
 const internalUtil = require('internal/util');
 const runInThisContext = require('vm').runInThisContext;
-const assert = require('assert').ok;
-const fs = require('fs');
 const path = require('path');
 const internalModuleReadFile = process.binding('fs').internalModuleReadFile;
 const internalModuleStat = process.binding('fs').internalModuleStat;
 
 
+function fs() {
+  if (fs.cached === undefined)
+    fs.cached = require('fs');
+  return fs.cached;
+}
+fs.cached = undefined;
```

Comment:
Not strictly necessary, no, but it's good for clarity and performance reasons (only one hidden class to deal with inside the function.)